modules/core/src/smithy4s/http/matchPath.scala

```diff
@@ -22,20 +22,6 @@ import PathSegment._
 
 object matchPath {
 
-  def make(str: String): Array[String] =
```

Comment:
We can probably leave it in with a deprecation and/or `private` to satisfy BC

Reply:
yeah let's do that, since it's a bugfix I'd be keen to ship it in a patch release